opentok/opentok.py

```diff
@@ -909,11 +926,17 @@ def set_archive_layout(self, archive_id, layout_type, stylesheet=None):
 
         :param String stylesheet optional: CSS used to style the custom layout.
         Specify this only if you set the type property to 'custom'
+
+        :param String screenshare_type optional: Layout to use for screenshares. Must
+        set 'layout_type' to 'bestFit'
         """
         payload = {
             "type": layout_type,
         }
 
+        if screenshare_type is not None:
+            payload["screenshareType"] = screenshare_type
```

Comment:
More pass-through style stuff here - is I take it there's no pre-flight validation precedent in this SDK - if so I don't think there's anything else to do here - if there is I'd at least check the `layout_type` to ensure that it's `bestFit` as that's the requirement.

Reply:
For right now I'm just leaving it as pass-through, yes.